src/plugins/data_view_editor/public/plugin.tsx

```diff
@@ -63,6 +67,25 @@ export class DataViewEditorPlugin
       userPermissions: {
         editDataView: () => dataViews.getCanSaveSync(),
       },
+      /**
+       * Helper method to generate a new data view editor service.
+       * @returns DataViewEditorService
+       */
+      dataViewEditorServiceFactory: ({
```

Comment:
This functionality should be provided async with bundle splitting

Reply:
Updated here [`4043bb7` (#162853)](https://github.com/elastic/kibana/pull/162853/commits/4043bb7deddcdee82a01bded653ddb94a60d31d0)